specs/00_main.rst

```diff
@@ -0,0 +1,31 @@
++--------+-----------------+
+| issue  | title           |
++--------+-----------------+
+| 81_    | Address Book    |
++--------+-----------------+
+
+.. sectnum::
+
+.. _81: https://github.com/gnosis/safe/issues/81
+
+
+Address Book
+============
+
+Problem Definition
+------------------
+
+- User can recognize names better than full addresses
+- User wants to associate addresses to names
+- User can see the address book entries and their details
+- The address book is not synced (ie.: it is part of the local storage only)
+
+Contents
+--------
+
+- `Address Book List`_
+- `Add Address`_
+
```

Comment:
I propose a different name: instead of `Address Book Entry` use `Contact` or `Card`

Missing (at least mention of these use cases):
- Displaying transactions with addresses from the address book
  - Changes to Transaction List
  - Changes to Transaction Details
- Send screen changes
  - Select a recipient from the address book
  - Search a recipient by name, by address
  - Suggest user adding an address if it's not in the address book?
- Review screen updates (recipient)
- Delete Address Book Entry
- Change Address Book Entry
- Share address from address book?
- What are the entry points to the address book?
- Do we want to have this safe's / this user's contact in the address book as well? (Similar to the standard mobile address book card "Me")
- @tschubotz Do we want to associate an icon with an address? not only text? 
- @tschubotz Do we want to associate multiple addresses to one Address Book Entry? (Contact)
- @tschubotz Do we want to sync local (in-app) Address Book with iOS Address Book?
- Do we want to also track recently used addresses and add them to the selection list in the Send screen?

Reply:
> Displaying transactions with addresses from the address book
Should that be part of the transaction spec?

>Send screen changes
Shouldn't that be part of the other specs too?

This PR is targeted to the list.

Reply:
I do agree with @fmrsabino here that the integration in other screens should not be part of this spec. Either we generate a separate spec (Address book integration) or it is part of the affected specs.

I would also like to think about if "delete", "change", "share" and "details" in this spec or in a different one.

I would not integrate with the address book (at least v1) and if we want it I would add it as an additional spec.

Reply:
In general I would say this spec should describe the feature as is on Android (regarding icons, multiple addresses, phone contacts and recently used addresses)

> Do we want to have this safe's / this user's contact in the address book as well? (Similar to the standard mobile address book card "Me")

This is actually the case on Android, but for me belongs to a different spec